src/Data/Vec/Properties.agda

```diff
@@ -695,6 +713,19 @@ map-replicate :  ∀ (f : A → B) (x : A) n →
 map-replicate f x zero = refl
 map-replicate f x (suc n) = P.cong (f x ∷_) (map-replicate f x n)
 
+transpose-replicate : ∀ {m n} (xs : Vec A m) → transpose (replicate {n = n} xs) ≡ map replicate xs
+transpose-replicate {n = zero} _ = P.sym (map-const _ [])
+transpose-replicate {n = suc n} xs = begin
+  transpose (replicate xs)
```

Comment:
I think best to lay this out in the usual style, even if it violates the line length (which is more of a convention anyway)